packages/salesforcedx-vscode-core/test/vscode-integration/orgPicker/orgList.test.ts

```diff
@@ -137,7 +137,7 @@ describe('Filter Authorization Info', async () => {
       )
     ];
     defaultDevHubStub.returns('test-devhub1@gmail.com');
-    getUsernameStub.returns(undefined);
+    getUsernameStub.returns('test-devhub1@gmail.com');
```

Comment:
This emulates getUsername returning the username back to the execution when there isn't an alias.